### PR TITLE
Smoothing

### PR DESCRIPTION
**Pull request by db2882**

### CODE REVIEW

File: climbARUnity/Assets/Climbing Hold/ClimbingHold.cs [hunk 1]
```diff
@@ -1,93 +1,149 @@
 ﻿using UnityEngine;
 using System.Collections;
 using System;
-
+using System.Threading;
```
Comment (db2882) [inline climbARUnity/Assets/Climbing Hold/ClimbingHold.cs:4]: Remove

File: climbARUnity/Assets/Climbing Hold/MenuHold.cs [hunk 2]
```diff
@@ -10,66 +10,65 @@ public class MenuHold : ClimbingHold
...
-    {
-        if (!ShouldRegisterHoldReleased(col))
+        if (grabbedState == States.Released && currentState == States.Grabbed)
         {
-            return;
-        }
+            grabbedState = States.Grabbed;
 
-        StopCoroutine(coroutine);
+            StartCoroutine(coroutine);
```
Comment (db2882) [inline climbARUnity/Assets/Climbing Hold/MenuHold.cs:26]: Indenting?

File: climbARUnity/ProjectSettings/EditorBuildSettings.asset [hunk 3]
```diff
@@ -11,7 +11,7 @@ EditorBuildSettings:
     path: Assets/MusicGame.unity
   - enabled: 1
     path: Assets/RocMan/RocManGamePlay.unity
-  - enabled: 1
+  - enabled: 0
```
Comment (db2882) [inline climbARUnity/ProjectSettings/EditorBuildSettings.asset:14]: Why this?

File: climbARUnity/Assets/Climbing Hold/MenuHold.cs (hunk 2 above)
Comment (patxu) [reply climbARUnity/Assets/Climbing Hold/MenuHold.cs:26]: no this looks good bc it's in the indent block. same as the line above, right?

**Pull request merged**